bioptim/interfaces/acados_interface.py

```diff
@@ -609,44 +607,6 @@ def __update_solver(self):
             else:
                 self.ocp_solver.set(self.acados_ocp.dims.N, "x", self.ocp.nlp[0].x_init.init[:, self.acados_ocp.dims.N])
 
-    def configure(self, options: SolverOptionsAcados = SolverOptionsAcados()):
-        """
-        Set some ACADOS options
-
-        Parameters
-        ----------
-        options: SolverOptionsAcados
-            The dictionary of options
-        """
-
-        options = options.__dict__
-
-        if "acados_dir" in options:
-            del options["acados_dir"]
-        if "cost_type" in options:
-            del options["cost_type"]
-        if "constr_type" in options:
-            del options["constr_type"]
-
-        if self.ocp_solver is None:
-            for key in options:
-                setattr(self.acados_ocp.solver_options, key, options[key])
-        else:
-            available_options = [
-                "nlp_solver_tol_comp",
-                "nlp_solver_tol_eq",
-                "nlp_solver_tol_ineq",
-                "nlp_solver_tol_stat",
-            ]
-            for key in options:
-                if key in available_options:
-                    short_key = key[11:]
-                    self.ocp_solver.options_set(short_key, options[key])
-                else:
-                    raise RuntimeError(
-                        f"[ACADOS] Only editable solver options after solver creation are :\n {available_options}"
-                    )
```

Comment:
This may cause a problem... The available options to modify the first time you call ACADOS are not the same as the subsenquent times.. This is the intent of this else (to return some options if it is the first time and some precise one otherwise... We can discuss the better way to implement this feature

Reply:
Maybe I missed something, but it seemed to work without it. Let's talk about it whenever you can.